### DISCUSSION TITLE
AeoTec Door Sensor 7 Pro and Multisensor Indicator CC Node Identify Parameters?

### DISCUSSION THREAD

**Discussion by asucrews** [zwave-js/zwave-js#4884] [Q&A]
Hello,

Does anyone know what the below Parameters are use for and how to use them? I can’t find any documentation on AeoTec support pages about them. I found the below link explain the CC but I don't really get what AeoTec trying to do.

In the Indicator CC
    Node Identify - On/Off Period
    Node Identify - On/Off Cycle Count
    Node Identify - On/Off Period: On Time

https://z-wave.me/manual/z-way/Command_Class_Reference.html#SECTION0016360000000000000000

**Comment by AlCalzone** [comment]
The node identify feature of the Indicator CC is required to be supported by newer devices. Its not terribly useful that they are represented as 3 different values since they should all be set together to identify the node (make its LED blink).

You can find more details in the Z-Wave specs: https://github.com/zwave-js/specs/blob/master/Z-Wave%20Specification%20AWG%20V1.0.pdf
